src/renderer/store/Namespace.js

```diff
@@ -29,6 +33,8 @@ export class Namespace {
     let _clusters = null;
     let _sshKeys = null;
     let _credentials = null;
+    let _proxies = null;
+    let _licenses = null;
```

Comment:
I think it'll be better for downstream code if we ensure that these properties are always arrays, and empty if none/unknown. If we don't know what they are, that's because we haven't fetched the namespace, and we won't have a Namespace class instance in that case anyway.

```suggestion
    let _clusters = [];
    let _sshKeys = [];
    let _credentials = [];
    let _proxies = [];
    let _licenses = [];
```